src/App.tsx

```diff
@@ -7,10 +7,11 @@ import usersFromServer from './api/users';
 import { User } from './types/User';
 import { Todo } from './types/Todo';
 
+import { TodoList } from './components/TodoList/index';
+
 function getUser(userId: number): User | null {
   const foundUser = usersFromServer.find(user => user.id === userId);
```

Comment:
Because you store here user object. Found(verb) we use only at the beginning of the function.
```suggestion
  const foundedUser = usersFromServer.find(user => user.id === userId);
```